code/datums/ai/monkey/monkey_subtrees.dm

```diff
@@ -76,7 +76,7 @@
 			controller.queue_behavior(/datum/ai_behavior/consume, pick(food_candidates))
 			return
 
-	if(prob(50))
+	if(!ismachinery(living_pawn.loc) && prob(50))
```

Comment:
Shouldn't it just be so that they can only perform this while on a turf? I don't see why they would ever need to be able to do this while inside of something unless they can pilot mechs or something weird that I'm not considering